### ISSUE TITLE
Consider adding EPA CAMPD facility inventory

### ISSUE THREAD

**Issue by dt-woods** [USEPA/standardizedinventories#158]
As described in https://github.com/USEPA/ElectricityLCI/issues/207, annual facility-level emissions from EPA's Clean Air Markets Program Data are desired for building a life cycle inventory for U.S. electricity generation. The EPA provide a web-based portal ([here](https://campd.epa.gov/data/custom-data-download)) and an [API](https://www.epa.gov/power-sector/cam-api-portal#/swagger/emissions-mgmt) to access this data. A method, `build_cems_df` is available to query the API for all the state-based facility-level annual emissions, [here](https://github.com/KeyLogicLCA/ElectricityLCI/blob/a2d4d4f32874a218372926d28a40d8e8383e4d36/electricitylci/cems_data.py#L138), which requires a user to register for an API key ([here](https://www.epa.gov/power-sector/cam-api-portal#/api-key-signup)). For harmonization with other facility-level data, it would be nice to have this a part of StEWI.

**Comment by bl-young** [comment]
is the json here: https://github.com/USEPA/ElectricityLCI/issues/207#issuecomment-1751075194 a pretty accurate estimate of what you anticipate the ultimate data looking like? If so, it would be quite similar to our eGRID facility data.

**Comment by bl-young** [comment]
Seems like a reasonable candidate, I'll ask @WesIngwersen to chime in.

**Comment by WesIngwersen** [comment]
Yes, that one has been on the radar for awhile. Not something our team can support immediately except through connection with the Clean Air Markets Division (CAMD) data team to discuss the ideal approach (also might be a more ideal way to update eGRID). Please email with further planning inquiries/plans.

**Comment by dt-woods** [comment]
> is the json here: [USEPA/ElectricityLCI#207 (comment)](https://github.com/USEPA/ElectricityLCI/issues/207#issuecomment-1751075194) a pretty accurate estimate of what you anticipate the ultimate data looking like? If so, it would be quite similar to our eGRID facility data.

Yes. That's the data format we're working with.
